yarn.lock

```diff
@@ -2308,26 +2308,12 @@
     glob-to-regexp "^0.3.0"
 
 "@mui/core@^5.0.0-alpha.54":
-  version "5.0.0-alpha.58"
-  dependencies:
-    "@babel/runtime" "^7.16.3"
-    "@emotion/is-prop-valid" "^1.1.1"
-    "@mui/utils" "^5.2.2"
-    "@popperjs/core" "^2.4.4"
-    clsx "^1.1.1"
-    prop-types "^15.7.2"
-    react-is "^17.0.2"
+  version "0.0.0"
```

Comment:
🤔  Not excatly sure what's going on here, this is what the lockfile ends up like after installing over the one that's on the master branch. (yarn v1.22.10)

Reply:
@siriwatknp @hbjORbj 
Tagging you to ask if you have any ideas as to why this [one particular test keeps failing](https://app.circleci.com/pipelines/github/mui-org/material-ui/61497/workflows/4497dfe5-0b08-4cd5-a21e-121270a22a1a/jobs/335353) for this PR. It'll be very helpful if you do!

Reply:
yeah, have you run `yarn`. I think the `yarn.lock` is not sync or updated.

Reply:
I've added `jarle` as a dependency manually to  `package.json` and then run`yarn` (doing it via `yarn add` seems to break)

For Sandpack - I am exploring it currently, although it might be more long term and this option might still be the one to go with till that is implementable